crates/re_ui/src/list_item.rs

```diff
@@ -363,6 +383,11 @@ impl<'a> ListItem<'a> {
         let desired_size = egui::vec2(desired_width, self.height);
         let (rect, mut response) = ui.allocate_at_least(desired_size, egui::Sense::click());
 
+        // handle dragging
+        if let Some(drag_id) = self.drag_id {
+            response = ui.interact(response.rect, drag_id, egui::Sense::drag());
```

Comment:
We should also set `CursorIcon` to `Grab`/`Grabbing`, like the demo at egui.rs does. This is an important [_affordance_](https://en.wikipedia.org/wiki/Affordance) that tells the user that "I can drag this".

Reply:
Added `Grabbing` cursor.

I'm unconvinced about _always_ having the `Grab` cursor when hovering over the blueprint tree. That is too busy and indicates that drag and drop is the _only_ possible operation. FWIW: Figma does none of that.